### ISSUE TITLE
Hyperv box 'ericmann/trusty64' does not exist anymore

### ISSUE THREAD

**Issue by roelveldhuizen** [Varying-Vagrant-Vagrants/VVV#1082]
It seems that the hyperv box, ```ericmann/trusty64```, is offline (See output bellow). Also cannot find it at the output url or on anyother references I found. 

```
$ vagrant up --provider=hyperv
Bringing machine 'default' up with 'hyperv' provider...
==> default: Verifying Hyper-V is enabled...
==> default: Box 'ericmann/trusty64' could not be found. Attempting to find and install...
    default: Box Provider: hyperv
    default: Box Version: >= 0
The box 'ericmann/trusty64' could not be found or
could not be accessed in the remote catalog. If this is a private
box on HashiCorp's Atlas, please verify you're logged in via
`vagrant login`. Also, please double-check the name. The expanded
URL and error message are shown below:

URL: ["https://atlas.hashicorp.com/ericmann/trusty64"]
Error: The requested URL returned error: 404 Not Found
```

**Comment by jeremyfelt** [comment]
Hi @roelveldhuizen, thanks for opening an issue. We'll need to get that box back up or find an alternative.

@ericmann Can you provide any more detail on this? :smile:

**Comment by ericmann** [comment]
We'll likely need to find an alternative. That box was badly out of date and was taken offline several weeks ago.

**Comment by benlumia007** [comment]
@ericmann 
How did you get Hyper-V to work, I've been trying to get to work on my Surface Pro 4. Any idea.

**Comment by ericmann** [comment]
First, you need a box. I had a Trusty build that I had configured on Hyper-V myself. But it was costing me a bunch of money to host, so I killed it late last year. If you do find a box, it'll work fine so long as you:
- have the Pro edition of Windows
- explicitly enable Hyper-V support (using "Turn Windows Features On/Off")

**Comment by Johnhhorton** [comment]
@ericmann can you add reference to an active box that works? While I can find one and apply it, it's difficult for new users to find out what happened when your removed box is still in the vagrantfile.

**Comment by ericmann** [comment]
@Johnhhorton unfortunately no. I have't used VVV in HyperV since last summer and haven't maintained or tested any Trusty builds under that VM since December at the latest.

**Comment by Johnhhorton** [comment]
@ericmann Thanks for the quick reply!  I'm coming from using a Docker setup for local development, but was running into too many issues.  I'm not extremely familiar with Vagrant and what would make a good build.  If you can provide any guidance (vague or specific), I'm sure it'll help others trying to solve this issue.

**Comment by ericmann** [comment]
When I made the Trusty build above I literally sat a MacBook and a Windows PC side by side and spun up fresh Trusty installs on each (Mac with VirtualBox, Windows with Hyper-V) so they'd have 100% parity.

The default Vagrant Hyper-V docs were particularly useful in keeping track of what to install as I went: https://www.vagrantup.com/docs/hyperv/boxes.html

Note: I built the original box a couple of _years_ ago, so my memory is foggy on all of the steps I took ... but those docs, and the docs for building base boxes in general, might get you started.

**Comment by ericmann** [comment]
FWIW, you might want to look at @johnpbloch's Trusty box as an alternative. https://atlas.hashicorp.com/johnpbloch/boxes/trusty64

**Comment by johnpbloch** [comment]
I haven't used vagrant in a while, so that box should come with the caveat that it's unsupported, i don't maintain it, will not be maintaining it, etc.

But hashicorp hosts it for free, so I'm also not going to be taking it offline. So if it works, use it. If it doesn't you'll have to figure out how to build your own hyper-v box. If that happens...

![anigif_eaa6a580d8aece464ad6ec5fd8670b68-0](https://cloud.githubusercontent.com/assets/446833/26178773/5554f09e-3b25-11e7-8cdb-426d0aebc2d9.gif)

**Comment by Johnhhorton** [comment]
@ericmann @johnpbloch Thanks guys.  I totally understand.  I ended up going with phawxby/trusty64 since it said "Standard Ubuntu 14.04 box with a large drive. Just a resized version of ericmann/trusty64"

**Comment by ericmann** [comment]
@Johnhhorton if that box is working, would you be up to submitting a PR to this repo swapping out the box definitions?

**Comment by Viper007Bond** [comment]
I'm not sure if other tickets referencing this one triggers notifications or not, so I'm manually pinging @Johnhhorton here again. I ran into this issue myself recently and would love for someone to provide a prebuilt box. :)

**Comment by Johnhhorton** [comment]
I ended up switching over to Docker for local development instead.
Otherwise I would have made a pull request. You should be able to try
modifying the config and adding boxes that you know exist until you find
one that works. I don't know what ericmann did to his box, but if it's
relatively standard then you should be able to find something else.

Good luck.

**Comment by Viper007Bond** [comment]
Thanks and no worries. I'm currently using Docker as well ( https://github.com/10up/wp-local-docker/ specifically ) but thought I'd experiment with VVV too.

**Comment by jeremyfelt** [comment]
Fixed in #1307 with the `phawxby/trusty64` box.
